From 8b3671f1ccabdc73a17276c059e6451026b26585 Mon Sep 17 00:00:00 2001
From: Enrico Scholz <enrico.scholz@sigma-chemnitz.de>
Date: Wed, 26 Nov 2008 14:13:18 +0100
Subject: [PATCH] Separate '-m' and the mode with a space when invoking 'install'

Some 'install' versions (e.g. this of OpenEmbedded) require the mode
to be a separate argument and don't understand the '-m0755' syntax:

  .../staging/x86_64-linux/usr/bin/install-sh: ./-m0755 does not exist.

Signed-off-by: Enrico Scholz <enrico.scholz@sigma-chemnitz.de>
---
 Makefile                     |    2 +-
 mkfs.ubifs/Makefile          |    2 +-
 ubi-utils/Makefile           |    2 +-
 ubi-utils/new-utils/Makefile |    2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 796c31b..8319599 100644
--- a/Makefile
+++ b/Makefile
@@ -96,7 +96,7 @@ $(BUILDDIR)/fectest: $(BUILDDIR)/fectest.o $(BUILDDIR)/crc32.o $(BUILDDIR)/fec.o
 
 install: ${TARGETS}
 	mkdir -p ${DESTDIR}/${SBINDIR}
-	install -m0755 ${TARGETS} ${DESTDIR}/${SBINDIR}/
+	install -m 0755 ${TARGETS} ${DESTDIR}/${SBINDIR}/
 	mkdir -p ${DESTDIR}/${MANDIR}/man1
 	gzip -9c mkfs.jffs2.1 > ${DESTDIR}/${MANDIR}/man1/mkfs.jffs2.1.gz
 	$(MAKE) -C $(BUILDDIR)/ubi-utils install
diff --git a/mkfs.ubifs/Makefile b/mkfs.ubifs/Makefile
index a327eee..f802bff 100644
--- a/mkfs.ubifs/Makefile
+++ b/mkfs.ubifs/Makefile
@@ -20,4 +20,4 @@ cscope:
 
 install: ${TARGETS}
 	mkdir -p ${DESTDIR}/${SBINDIR}
-	install -m0755 ${TARGETS} ${DESTDIR}/${SBINDIR}/
+	install -m 0755 ${TARGETS} ${DESTDIR}/${SBINDIR}/
diff --git a/ubi-utils/Makefile b/ubi-utils/Makefile
index 4cca316..63058e1 100644
--- a/ubi-utils/Makefile
+++ b/ubi-utils/Makefile
@@ -75,7 +75,7 @@ pfi2bin: pfi2bin.o peb.o error.o list.o crc32.o libubigen.o bootenv.o \
 
 install: ${TARGETS}
 	mkdir -p ${DESTDIR}/${SBINDIR}
-	install -m0755 ${TARGETS} ${DESTDIR}/${SBINDIR}/
+	install -m 0755 ${TARGETS} ${DESTDIR}/${SBINDIR}/
 	(cd perl && install ${PERLPROGS} ${DESTDIR}/${SBINDIR}/)
 
 uninstall:
diff --git a/ubi-utils/new-utils/Makefile b/ubi-utils/new-utils/Makefile
index ba4c20f..356fc96 100644
--- a/ubi-utils/new-utils/Makefile
+++ b/ubi-utils/new-utils/Makefile
@@ -66,7 +66,7 @@ clean:
 
 install: ${UTILS}
 	mkdir -p ${DESTDIR}/${SBINDIR}
-	install -m0755 ${UTILS} ${DESTDIR}/${SBINDIR}/
+	install -m 0755 ${UTILS} ${DESTDIR}/${SBINDIR}/
 
 uninstall:
 	for file in ${UTILS}; do \
-- 
1.6.0.4

